src/mesh.cpp

```diff
@@ -407,6 +407,13 @@ scene::IMesh* cloneMesh(scene::IMesh *src_mesh)
 
 scene::IMesh* convertNodeboxNodeToMesh(ContentFeatures *f)
 {
+	std::vector<aabb3f> boxes = f->node_box.fixed;
+	return convertNodeboxesToMesh (boxes, NULL);
+}
+
+scene::IMesh* convertNodeboxesToMesh(std::vector<aabb3f> &boxes, f32 *txc)
+{
+	bool use_default_txc = (txc != NULL) ? false : true;
```

Comment:
its not useless, i have to check that once before iterating over boxes. otherwise i would get default uv being used always after the 2nd box